src/assets/sass/_header.scss

```diff
@@ -0,0 +1,109 @@
+@media only screen and (min-width: 769px){
+    .header-container {
+        display: block;
+    }
+}
+.header-container .header-main {
+    color: #0b0c0c;
+    padding: 0.5rem 0px;
+    border-bottom: 6px solid #008938;
```

Comment:
For colors, we can use variables that we get from the DEFRA.

- $gov-defra-color-websafe - For #008938 - It is defined [here](https://github.com/DEFRA/ncea-frontend/blob/6c084039805b64ed173bf51a1829d3c48c1ced00/src/assets/sass/_variables.scss#L1).
-  $govuk-text-colour or govuk-colour("black") for #0b0c0c
- govuk-colour("green") for #00703c
- govuk-colour("white") for #FFF

Reply:
Fixed